resteasy-jaxrs/src/main/java/org/jboss/resteasy/plugins/providers/RegisterBuiltin.java

```diff
@@ -52,7 +59,7 @@ public static void register(ResteasyProviderFactory factory)
       }
    }
 
-   public static void registerProviders(ResteasyProviderFactory factory) throws Exception
+   public static void registerProviders(ResteasyProviderFactory factory, Set<String> disabledProviders) throws Exception
```

Comment:
minor thing, but this need the old method to be retained too, as you did for the `register(ResteasyProviderFactory factory)` one above (full backward compatibility needs)

Reply:
Done